plugins/woocommerce/src/Admin/Features/ShippingPartnerSuggestions/DefaultShippingPartners.php

```diff
@@ -0,0 +1,250 @@
+		$row_layout_features = array(
+			array(
+				'icon'        => $asset_base_url . 'timer.svg',
+				'title'       => __( 'Save time', 'woocommerce' ),
+				'description' => __(
+					'Automatically import order information to quickly print your labels.',
+					'woocommerce'
+				),
+			),
+			array(
+				'icon'        => $asset_base_url . 'discount.svg',
+				'title'       => __( 'Save money', 'woocommerce' ),
+				'description' => __(
+					'Shop for the best shipping rates, and access pre-negotiated discounted rates.',
+					'woocommerce'
+				),
+			),
+			array(
+				'icon'        => $asset_base_url . 'star.svg',
+				'title'       => __( 'Wow your shoppers', 'woocommerce' ),
+				'description' => __(
+					'Keep your customers informed with tracking notifications.',
+					'woocommerce'
+				),
+			),
+		);
+
+		$check_icon = $asset_base_url . 'check.svg';
+
+		return array(
+			array(
+				'name'              => 'ShipStation',
+				'slug'              => 'woocommerce-shipstation-integration',
+				'description'       => __( 'Powerful yet easy-to-use solution:', 'woocommerce' ),
+				'layout_row'        => array(
+					'image'    => $asset_base_url . 'shipstation-row.svg',
+					'features' => $row_layout_features,
+				),
+				'layout_column'     => array(
+					'image'    => $asset_base_url . 'shipstation-column.svg',
+					'features' => array(
+						array(
+							'icon'        => $check_icon,
+							'description' => __(
+								'Print labels from Royal Mail, Parcel Force, DPD, and many more',
+								'woocommerce'
+							),
+						),
+						array(
+							'icon'        => $check_icon,
+							'description' => __(
+								'Shop for the best rates, in real-time',
+								'woocommerce'
+							),
+						),
+						array(
+							'icon'        => $check_icon,
+							'description' => __( 'Connect selling channels easily', 'woocommerce' ),
+						),
+						array(
+							'icon'        => $check_icon,
+							'description' => __( 'Advance automated workflows', 'woocommerce' ),
+						),
+						array(
+							'icon'        => $check_icon,
+							'description' => __( '30-days free trial', 'woocommerce' ),
+						),
+					),
+				),
+				'learn_more_link'   => 'https://wordpress.org/plugins/woocommerce-shipstation-integration/',
+				'is_visible'        => array(
+					self::get_rules_for_countries( array( 'AU', 'CA', 'GB' ) ),
+				),
+				'available_layouts' => array( 'row', 'column' ),
+			),
+			array(
+				'name'              => 'Skydropx',
+				'slug'              => 'skydropx-cotizador-y-envios',
+				'layout_row'        => array(
+					'image'    => $asset_base_url . 'skydropx-row.svg',
+					'features' => $row_layout_features,
+				),
+				'description'       => '',
+				'learn_more_link'   => 'https://wordpress.org/plugins/skydropx-cotizador-y-envios/',
+				'is_visible'        => array(
+					self::get_rules_for_countries( array( 'MX', 'CO' ) ),
+				),
+				'available_layouts' => array( 'single' ),
```

Comment:
Should it be "row" layout?

Reply:
@chihsuan Thank you for catching it 🙏

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/37155/commits/302ffe0e2dbb79dd905e5cf3956a253d8afa4275